app/liquid/liquid_renderer.rb

```diff
@@ -116,7 +117,12 @@ def thermometer
   end
 
   def call_tool_data
-    plugin_data.deep_symbolize_keys[:plugins][:call_tool]
+    plugin_data.deep_symbolize_keys[:plugins][:call_tool].map do |data|
+      %i[target_phone target_country_code target_phone_extension target_name target_title].each do |key|
```

Comment:
I don't quite follow... CallCreator is used at the endpoint that actually makes the call. Though we do stuff the parameters in two places, I think that calling into a service used in a different place to stuff the params seems odd. I don't quite understand about ensuring the objects always have the same shape, but I'd like to understand.